### PR TITLE
inspect multiple boxes and display them in a single figure

### LINKED ISSUE #124: [Feature] Support inputting multiple points (in a list) with PrimBox.inspect()
Instead of only inspecting one point, it would be neat if you can pass multiple points into the [`inspect()`](https://github.com/quaquel/EMAworkbench/blob/master/ema_workbench/analysis/prim.py#L405) function of an `analysis.prim.PrimBox`. The expected behavior could be:

- [x] For `PrimBox.inspect(i=list, style="table")`, a dataframe with a row for each point will be provided.
- [ ] For `PrimBox.inspect(i=list, style="graph")`, either Figure with a subplot for each point could be provided, or a single graph with the multiple points compared to each other (each point with a different color bars).

This will allow to more quickly inspect and compare multiple points on the peeling trajectory.

Also I noted that the inspect function prints a lot of data without returning it. I think it's nicer and more expected to return the values/tables/graphs, and let the user print them. That way you can use them in further analysis if you want.

### PR DESCRIPTION
**Pull request by quaquel**
closes #124

Adds an optional keyword argument ax to `PrimBox.inspect` to be used in conjunction with `style='graph'`. Ax can be a single Axes or a list of Axes instances of the same length as `i`.  If used, each box in i is plotted in separate Axes.

Rather that try to solve the axes layout as suggested in #124, it is left to the user to setup the figure to their liking. See [sd_prim_flu.py](https://github.com/quaquel/EMAworkbench/blob/ec46ce60836a7b1533e4eb177239a0c251f2c74a/ema_workbench/examples/sd_prim_flu.py) for an example usage.

### CODE REVIEW

**Comment by EwoutH** [comment]
Thanks for implementing this, I will try to review it this week.

**Comment by quaquel** [comment]
Just a simple reminder that it would be great if you could take a quick look at this so I can merge it.

**Comment by EwoutH** [comment]
Thanks, I put it on my list for Monday.

To give some context, I'm now really in a hard push to get my thesis started. As always, the things I find most interesting are also the hardest and biggest. Then this requires to get a full overview of the system to properly identify the gaps - and then selecting a feasonable one. So that consumes most of my energy right now.

Also, maybe it would be good to talk about my formal role in the workbench maintenance again. When I was TAing in EPA1361 and after that working on the MPIEvaluator is was a relatively low effort to do it on the side (and a great learning experience - it still is), I now notice takes a bit more effort to switch to it.

File: test/test_analysis/test_scenario_discovery_util.py [hunk 1]
```diff
@@ -185,7 +185,8 @@ def test_plot_box(self):
 
         qp_values = {"a": [0.05, 0.9], "c": [0.05, -1]}
 
-        sdutil.plot_box(boxlim, qp_values, box_init, restricted_dims, 1, 1)
+        fig, ax = plt.subplots()
+        sdutil.plot_box(boxlim, qp_values, box_init, restricted_dims, 1, 1, ax)
```
Comment (EwoutH) [inline test/test_analysis/test_scenario_discovery_util.py:189]: Could you add another test here with multiple boxes and axes?

**Review by EwoutH — APPROVED** [review]
Looks great! I really like the warnings for catching invalid input.

Only thing I would like to see is another test with multiple boxes and axes, further everything looks good to me!

**Pull request merged**